Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12-alpine AS build
+# packages needed for node-gyp and dupe-krill
+RUN apk add --update --no-cache python make g++ rust cargo
+# use a non-root user to make commands such as npm post-install scripts safer
+USER node
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/.npm-global/bin:/home/node/.cargo/bin
+# RUN cargo install dupe-krill
+# copy by default will make the root user own the files, we want the node user to own the files
+COPY --chown=node:node package*.json .snyk /home/node/app/
+WORKDIR /home/node/app
+RUN npm install
```

Comment:
When I build this Dockerfile locally, this installs `polyfill-library-3.27.4`, but `node_modules/polyfill-library-3.27.4/polyfills/__dist/` doesn't exist, which causes `npm run build` to fail.

When I install it on my local machine, the `__dist` folder is present and the build succeeds.

Reply:
I think the problem is something to do with system permissions. `npm install --unsafe-perm` seems to fix it!

Reply:
I just tried again with your latest and no local changes. It works. I don't know what I was doing wrong. Resolved. 👍